### PR TITLE
DEV-4475: Log new line as <enter>

### PR DESCRIPTION
**Pull request by linaza-siklu**
**Description:**
      This Pull request supports Log new line as enter, In all of the script’s logs, log \n as 'enter'.
      changes in DeviceSetup.py, saved output string and changed '\n' to 'enter'

**Tests :** 
   tested log output and got:
        python3 main.py devices/tg-n366.yaml --command '/usr/bin/telnet 10.40.1.122 3001' --io-log device_console_central_node.log
       INFO:DeviceSetup:Starting state InitState...
       INFO:DeviceSetup:Running state actions...
       INFO:DeviceSetup:Writing output: "enter"

Jira Issue No'
DEV-4475

### CODE REVIEW

File: setup_script/DeviceSetup.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import logging
 from interfaces.AbstractDeviceSetup import AbstractDeviceSetup
 from interfaces.AbstractDeviceIO import AbstractDeviceIO
-
 from models.pydantic.device_model import Actions, Match, Input, State, Output, ErrorState, FinalState
 
+
```
Comment (dima1308) [inline setup_script/DeviceSetup.py:6]: why logger should be global? Why not initialize it on main and pass it as a parameter to the device setup class?

Comment (linaza-siklu) [reply setup_script/DeviceSetup.py:6]: @shmuelhazan should i change it as Dima suggested?

Comment (shmuelhazan) [reply setup_script/DeviceSetup.py:6]: @linaza-siklu Yes.

Comment (linaza-siklu) [reply setup_script/DeviceSetup.py:5]: Done

**Comment by dima1308** [comment]
looks good to me

**Review by shmuelhazan — APPROVED** [review]

**Pull request merged**